core/shared/src/main/scala/io/circe/Decoder.scala

```diff
@@ -1,11 +1,13 @@
 package io.circe
 
+import java.nio.ByteBuffer
+
 import cats.Monad
 import cats.data._
 import cats.std.list._
 import cats.syntax.cartesian._
 import cats.syntax.functor._
-import java.util.UUID
+import java.util.{ Base64, UUID }
```

Comment:
Ugh, yeah, I don't think we should drop Java 7 support quite yet, so it looks like if we want these instances in core, we'd need to use the `javax.xml.bind.DatatypeConverter` approach, which isn't going to work on Scala.js (where it looks like we'd need to use [`btoa`](https://groups.google.com/forum/#!msg/scala-js/En8WL-WIY_Q/147WEVIIOKAJ)).

I'd personally prefer to have these instances in core, since `ByteBuffer` itself isn't Java 8-specific. Maybe we could switch to `DatatypeConverter` for this PR, and I can follow up with a Scala.js-specific implementation?